internal/voting/voting.go

```diff
@@ -23,9 +23,10 @@ const (
 func InitializeVoteStore(ctx context.Context, db sql.DB) error {
 	upgradeFns := map[int64]versioning.UpgradeFunc{
 		0: initTables,
+		1: upgradeV0ToV1,
 	}
 
-	err := versioning.Upgrade(ctx, db, votingSchemaName, upgradeFns, voteStoreVersion)
+	err := versioning.Upgrade(ctx, db, VotingSchemaName, upgradeFns, voteStoreVersion)
```

Comment:
Think we need to bump `voteStoreVersion = 1` (well, 2 after rebase) to actually make it run the upgrades.  Probably that says that const is pointless when we can just do `len(upgradeFns)` but that's how we work for now.